src/remote_adapter/remote_adapter_client/lua_lib/sdl_remote_adapter_client_lua_wrapper.h

```diff
@@ -1,8 +1,8 @@
 
 extern "C" {
-#include "lua.h"
-#include "lualib.h"
-#include "lauxlib.h"
+#include <lua5.2/lua.h>
```

Comment:
@aderiabin I think better to avoid such prefix lua5.2 before include
see recommendations:
https://cmake.org/cmake/help/v3.6/module/FindLua.html

Reply:
@EKuliiev Applied (see 16996cc5aa4b9fa3938a0dd47c37e1f8172677a3)